diff --git a/node_modules/kysely/package.json b/node_modules/kysely/package.json
--- a/node_modules/kysely/package.json
+++ b/node_modules/kysely/package.json
@@ -63,6 +63,11 @@
       "require": "./dist/cjs/helpers/sqlite.js",
       "default": "./dist/cjs/helpers/sqlite.js"
     }
+    "./migration": {
+      "import": "./dist/esm/migration/index.js",
+      "require": "./dist/cjs/migration/index.js",
+      "default": "./dist/cjs/migration/index.js"
+    }
   ],
   "sideEffects": false,
diff --git a/node_modules/kysely/dist/cjs/migration/index.d.ts b/node_modules/kysely/dist/cjs/migration/index.d.ts
new file mode 100644
--- /dev/null
+++ b/node_modules/kysely/dist/cjs/migration/index.d.ts
@@ -0,0 +1,2 @@
+export * from './file-migration-provider.js';
+export * from './migrator.js';
diff --git a/node_modules/kysely/dist/cjs/migration/index.js b/node_modules/kysely/dist/cjs/migration/index.js
new file mode 100644
--- /dev/null
+++ b/node_modules/kysely/dist/cjs/migration/index.js
@@ -0,0 +1,11 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.FileMigrationProvider = exports.DEFAULT_MIGRATION_TABLE = exports.DEFAULT_MIGRATION_LOCK_TABLE = exports.DEFAULT_ALLOW_UNORDERED_MIGRATIONS = exports.MIGRATION_LOCK_ID = exports.NO_MIGRATIONS = void 0;
+var migrator_js_1 = require("./migrator.js");
+Object.defineProperty(exports, "DEFAULT_MIGRATION_LOCK_TABLE", { enumerable: true, get: function () { return migrator_js_1.DEFAULT_MIGRATION_LOCK_TABLE; } });
+Object.defineProperty(exports, "DEFAULT_MIGRATION_TABLE", { enumerable: true, get: function () { return migrator_js_1.DEFAULT_MIGRATION_TABLE; } });
+Object.defineProperty(exports, "DEFAULT_ALLOW_UNORDERED_MIGRATIONS", { enumerable: true, get: function () { return migrator_js_1.DEFAULT_ALLOW_UNORDERED_MIGRATIONS; } });
+Object.defineProperty(exports, "MIGRATION_LOCK_ID", { enumerable: true, get: function () { return migrator_js_1.MIGRATION_LOCK_ID; } });
+Object.defineProperty(exports, "NO_MIGRATIONS", { enumerable: true, get: function () { return migrator_js_1.NO_MIGRATIONS; } });
+var file_migration_provider_js_1 = require("./file-migration-provider.js");
+Object.defineProperty(exports, "FileMigrationProvider", { enumerable: true, get: function () { return file_migration_provider_js_1.FileMigrationProvider; } });
diff --git a/node_modules/kysely/dist/esm/migration/index.js b/node_modules/kysely/dist/esm/migration/index.js
new file mode 100644
--- /dev/null
+++ b/node_modules/kysely/dist/esm/migration/index.js
@@ -0,0 +1,3 @@
+/// <reference types="./migrator.d.ts" />
+export * from './file-migration-provider.js';
+export * from './migrator.js';